### DISCUSSION TITLE
View and select program history

### DISCUSSION THREAD

**Discussion by maximesa** [astashov/liftosaur#145] [Ideas]
The ability to view a program version history and eventually rollback or fork a specific version

**Comment by astashov** [comment]
That actually exists - if you're logged in and editing your program in the web editor, on the right there's "Versions" link.

**Reply by maximesa** [reply]
Excellent I'll check the web,  I screwed up my program on mobile when I was changing the day and the program hadn't been updated yet, couldn't find versioning on mobile but if it's present on web I think that's enough
